crates/turborepo-lib/src/config/turbo.rs

```diff
@@ -92,7 +91,7 @@ struct RawTaskDefinition {
     #[serde(skip_serializing_if = "Option::is_none")]
     outputs: Option<Vec<String>>,
     #[serde(skip_serializing_if = "Option::is_none")]
-    output_mode: Option<TaskOutputMode>,
```

Comment:
Realized that `TaskOutputMode` is redundant since we already have it in `cli` as `OutputLogsMode`